modules/assisted-installer-assisted-installer-prerequisites.adoc

```diff
@@ -0,0 +1,62 @@
+// This is included in the following assemblies:
+//
+// installing-on-prem-assisted.adoc
+:_content-type: CONCEPT
+
+[id='assisted-installer-prerequisites_{context}']
+= Assisted Installer prerequisites
+
+Successful installation with the {ai-full} requires ensuring the following prerequisites. The installer will validate these prerequisites before installation.
+
+== Hardware
+
+For control plane nodes or the {sno} node, nodes must have at least:
+
+* 8 CPU cores
+* 15.00 GiB RAM
+* 100 GB storage
+* 10ms write speed or less for etcd `wal_fsync_duration_seconds`
+
+For worker nodes, each node must have at least:
+
+* 2 CPU cores
+* 8.00 GiB RAM
+* 100 GB storage
+
+== Networking
+
+The network must have:
+
+* A DHCP server unless using static IP addressing.
+* A base domain name. The base domain must have:
+  - A DNS A/AAAA record for `<cluster_name>.<base_domain>`. You must ensure that there is no wildcard, such as `*.<cluster_name>.<base_domain>`, or the installation will not proceed.
```

Comment:
I don't recall a requirement for `<cluster_name>.<base_domain>` resolution :thinking: